libs/ngrx-toolkit/package.json

```diff
@@ -9,10 +9,9 @@
   "peerDependencies": {
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
-    "@ngrx/signals": "^17.0.0"
-  },
-  "dependencies": {
```

Comment:
I think that nx might add this one automatically upon build? Can you check?
Here's more information: https://github.com/softarc-consulting/sheriff/commit/6f80696f9b6d8d6e76af40e445afa661180cf495

Reply:
There was once a setting to keep dependencies up to date during build but it was removed in a [v17 migration](https://github.com/nrwl/nx/blob/master/packages/js/src/migrations/update-17-0-0/remove-deprecated-build-options.ts#L28).

The `@nx/dependency-checks` rule provides fixers for missing or unused but defined dependencies - so it could be cleaned up automatically.

Reply:
Okay, then we might check that in another issue. Let's leave it for now.